cogs/time_tracking.py

```
@@ -1,96 +1,136 @@
+
+            gemini_client = self.bot.get_gemini_client(
+                env.GEMINI_STUDY_ENCOURAGEMENT_INSTRUCTION
+            )
+            status, text = await gemini_client.create_gemini_message(
+                f"공부시간:{total_minutes}분"
+            )
+
+            await alert_channel.send(
+                end_study_message(member.mention, minutes, total_minutes, text, status)
+            )
+        except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)